interna/crowdfund/tests/test_views.py

```diff
@@ -52,3 +53,34 @@ def test_status_codes(client, method, url, pk, user, status_code):
     reversed_url = reverse(url) if pk is None else reverse(url, kwargs={'pk': pk})
     response = getattr(client, method.lower())(reversed_url)
     assert response.status_code == status_code
+
+
+@pytest.mark.django_db
+def test_pledge_name_and_email(client):
+    prj = mommy.make(models.Project)
+    path = '/crowdfund/projects/{}/'.format(prj.pk)
+    response = client.get(path)
+    assert response.status_code == 200
+
+    # The name input looks like this <input name="name" value="Test" maxlength="100"
+    # class="textinput textInput form-control" required="" id="id_name" type="text">
+    # value should be missing if last_pledge_name isn't set
+    match = re.search(r'value="([^"]*)".*id="id_name"', response.content.decode())
```

Comment:
Since this exact RegEx is used twice, it could be extracted into a variable (maybe even with `re.compile`).

Reply:
Done.